test/featuretransforms.jl

```diff
@@ -0,0 +1,191 @@
+@testset "FeatureTransforms" begin
+    M1 = [0.0 1.0; 1.0 2.0; -0.5 0.0]
+    M2 = [-2.0 4.0; 3.0 2.0; -1.0 -1.0]
+    M3 = [0.0 1.0; -1.0 0.5; -0.5 0.0]
+    M4 = [0.5 -1.0; -5.0 -2.0; 0.0 1.0]
+    ds = KeyedDataset(
+        flatten([
+            :train => [
+                :load => KeyedArray(M1; time=1:3, loc=[:x, :y]),
+                :price => KeyedArray(M2; time=1:3, id=[:a, :b]),
+            ],
+            :predict => [
+                :load => KeyedArray(M3; time=1:3, loc=[:x, :y]),
+                :price => KeyedArray(M4; time=1:3, id=[:a, :b]),
+            ]
+        ])...
+    )
+
+    @testset "transform" begin
+        @test is_transformable(ds)
+    end
+
+    # TODO: use fake Transforms
```

Comment:
What are fake transforms?

Reply:
Forthcoming as part of test utils in in FeatureTransforms. See this POC PR https://github.com/invenia/FeatureTransforms.jl/pull/77/files#diff-4c5e126be8af5fe14f9784e4cedac0f729e29553a4fc76c5ca47fbd1c7e0a4d8R1
(Glenn is breaking up into multiple PRs at the moment)